assets/src/edit-story/app/media/media3p/typedefs.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * common media typedefs =================================================
+ */
+
+/**
+ * TODO(#3431) Move this to a media/typedef.js file as this is common for local
+ * as well as media3p.
+ *
+ * @typedef Media
+ * @property {Object} attribution
+ * @property {string} creationDate
+ * @property {number} height
+ * @property {number} width
+ * @property {string} id
+ * @property {number} length
+ * @property {*} lengthFormatted
+ * @property {boolean} local
+ * @property {string} mimeType
+ * @property {string} poster
+ * @property {string} posterId
+ * @property {Object} sizes
```

Comment:
'sizes' is a map that has a certain structure that can be captured in a type.

Reply:
Can I punt this to the next PR when i add a media/typedefs.js ?

Reply:
TODO next PR (based on offline discussion)